Integrations/MISP_V2/MISP_V2.py

```diff
@@ -728,7 +770,7 @@ def get_urls_events():
 
 
 def check_url(url):
-    response = MISP.search(value=url, type_attribute='url')
+    response = filter_misp_response(MISP.search(value=url, type_attribute='url'))
```

Comment:
the filter should affect the context only and not the `raw_response`.
`raw_response` should always be the original response from the 3rd party product.